### ISSUE TITLE
`quarkus-amazon-lambda-http`: can't build a Fast-JAR

### ISSUE THREAD

**Issue by nicolasduminil** [quarkiverse/quarkus-amazon-services#1298]
Hello,

With Quarkus 3.11, trying to perform a build containing a dependency to the `quarkus-amazon-lambda-http` extension, systematically creates a thin JAR instead a fast one.

To reproduce:

    $ git clone https://github.com/nicolasduminil/quarkus-test-case.git
    $ cd quarkus-test-case
    $ mvn package

The following message will appear in the log file:

    [INFO] [io.quarkus.deployment.pkg.steps.JarResultBuildStep] Building thin jar: /home/nicolas/quarkus-test-case/target/quarkus-test-case-runner.jar

Now comment out the following in the `pom.xml` file:

        

and build again:

    $ mvn clean package

The following message will be logged:

    [INFO] --- quarkus:3.11.0:build (default) @ quarkus-test-case ---

In conclusion, without the `quarkus-amazon-lambda-http` extension, the Fats-JAR is generated, as expected, however as soon as this dependency is added, only a thin JAR is generated, even if the `-Dquarkus.package.jar.type=fast-jar` option is used.

How could I avoid this behavior ? Many thanks in advance for your help.

Nicolas

**Comment by scrocquesel** [comment]
This seems to be by design. A deployment step was added specifically to force the legacy thin jar when fast-jar was introduced three years ago.  https://github.com/quarkusio/quarkus/pull/14833

If you want more information on why and if it can be changed, you should ask at quarkus core.

**Comment by nicolasduminil** [comment]
Could the issue be transferred please to whomever it concerns, such that to avoid this systematic round-trip between  teams and to the issuer to register several times the same issue ?

**Comment by maxandersen** [comment]
@nicolasduminil afaik GitHub doesn't allow for copy/transfer of issues between different organizations thus unfortunately no better way than recreate them manually.
